util/Pos/Util/Util.hs

```diff
@@ -91,34 +98,47 @@ import           Data.Time.Units (Microsecond, toMicroseconds)
 import qualified Ether
 import           Ether.Internal (HasLens (..))
 import qualified Formatting as F
+import           GHC.TypeLits (ErrorMessage (..))
 import qualified Language.Haskell.TH as TH
 import qualified Prelude
 import           Serokell.Util (listJson)
 import           Serokell.Util.Exceptions ()
 import           System.Wlog (LoggerName, WithLogger, logDebug, logError, logInfo, usingLoggerName)
 import qualified Text.Megaparsec as P
 
+----------------------------------------------------------------------------
+-- Cool stuff
+----------------------------------------------------------------------------
+
+type f ~> g = forall x. f x -> g x
+
 ----------------------------------------------------------------------------
 -- Exceptions/errors
 ----------------------------------------------------------------------------
 
 maybeThrow :: (MonadThrow m, Exception e) => e -> Maybe a -> m a
 maybeThrow e = maybe (throwM e) pure
 
--- | Throw exception or return result depending on what is stored in 'Either'
+-- | Throw exception (in 'MonadThrow') or return result depending on
+-- what is stored in 'Either'
 eitherToThrow
     :: (MonadThrow m, Exception e)
     => Either e a -> m a
 eitherToThrow = either throwM pure
 
+-- | Throw error (in 'MonadError e') or return result depending on
+-- what is stored in 'Either'
+eitherToMonadError
```

Comment:
Let's name this function `liftEither`, because it was added in `mtl-2.2.2` with this name, and we can remove it from our utils when we use newer `mtl`.